### DISCUSSION TITLE
Abandoned project?

### DISCUSSION THREAD

**Discussion by Mishasama** [rohanb10/snoozz-tab-snoozing#118] [Q&A]
@rohanb10 @vegeta897 
It seems that no one is maintaining it for a long time.

**Comment by vegeta897** [comment]
Seems that way. My only contribution was [a small PR](https://github.com/rohanb10/snoozz-tab-snoozing/pull/75), but I don't think it ever made it into the production build.

Hoping to see the project picked up again as I still use it and there were a lot of great ideas posted in the issues, but I understand the nature of solo-dev free open source projects.

**Reply by Mishasama** [reply]
> Seems that way. My only contribution was [a small PR](https://github.com/rohanb10/snoozz-tab-snoozing/pull/75), but I don't think it ever made it into the production build.
> 
> Hoping to see the project picked up again as I still use it and there were a lot of great ideas posted in the issues, but I understand the nature of solo-dev free open source projects.

It seems someone picked up the Tab Snooze. Maybe it's no reason to continue this project?

**Comment by loup1818** [comment]
Wow, still no reply since 2022?!
